src/health/lib/services/elastic.js

```diff
@@ -9,7 +9,12 @@ const logger = require('../logger');
 
 const isProd = (nodeEnv === 'production');
 
-const pingElasticWithClient = async () => {
+/**
+ * Ping elastic service.
+ *
+ * @returns {Promise<boolean>}
```

Comment:
The function can return a string. Either add `string` to the return types or make the function return only `boolean`